src/app/core/store/shopping/products/products.effects.ts

```diff
@@ -109,6 +111,26 @@ export class ProductsEffects {
     )
   );
 
+  loadProductPricesAfterProductSuccess$ = createEffect(() =>
+    this.actions$.pipe(
+      ofType(loadProductSuccess),
+      mapToPayloadProperty('product'),
+      mapToProperty('sku'),
+      whenTruthy(),
+      map(sku => loadProductPrices({ skus: [sku] }))
+    )
+  );
+
+  loadProductReviewsAfterProductSuccess$ = createEffect(() =>
```

Comment:
loading of the product review should be triggered by the product-reviews. component with the help of a facade method